src/main/java/com/unichristus/libraryapi/controller/UserController.java

```diff
@@ -0,0 +1,64 @@
+package com.unichristus.libraryapi.controller;
+
+import com.unichristus.libraryapi.model.dto.UserDTO;
+import jakarta.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/users")
```

Comment:
Aqui não tem necessidade do /api:
```suggestion
@RequestMapping("users")
```